### ISSUE TITLE
Keeps crashing on start up.

### ISSUE THREAD

**Issue by Lapiseq** [Chocohead/OptiFabric#581]
I think it's because of Iris, does anyone know why it happens?

net.fabricmc.loader.impl.FormattedException: net.fabricmc.loader.impl.discovery.ModResolutionException: Mod resolution encountered an incompatible mod set!
A potential solution has been determined:
	 - Remove mod 'OptiFabric' (optifabric) 1.12.10 (C:\Users\my name\AppData\Roaming\.minecraft\iris-reserved\1.18.1\optifabric-1.12.10 (1).jar).
Unmet dependency listing:
	 - Mod 'Iris' (iris) 1.1.3 is incompatible with any version of mod 'OptiFabric' (optifabric), but a matching version is present: 1.12.10!
		 - Mod 'OptiFabric' (optifabric) 1.12.10 is being loaded from C:\Users\my name\AppData\Roaming\.minecraft\iris-reserved\1.18.1\optifabric-1.12.10 (1).jar

**Comment by Zeraxon** [comment]
yes it is because of iris. iris is not needed since optifine comes with shader support.

**Comment by Chris6ix** [comment]
@Lapiseq 
I really dont know if you are trolling or what...
If you are serious, OptiFine and Iris are inherently completely incompatible with each other because they do the same thing (Shader support)

Pls, just pls READ BEFORE YOU DOWNLOAD...
Dont tell me you just download random stuff without even knowing shit about them just because the name sounds good or something?

You can close this issue as this is not even an issue...

**Comment by UnderseaKnight** [comment]
No its a real issue. I experience it with newer versions and have had to use knockoffs like sodium that suck in comparison to forge. It crashes on startup.
